tunnel/tunnel.go

```diff
@@ -383,10 +385,22 @@ func (t *Tunnel) keepAlive() {
 }
 
 func sshClientConfig(server Server) (*ssh.ClientConfig, error) {
+	var signers []ssh.Signer
+
 	signer, err := server.Key.Parse()
 	if err != nil {
 		return nil, err
 	}
+	signers = append(signers, signer)
+
+	agentAddr := os.Getenv("SSH_AUTH_SOCK")
```

Comment:
I am having mixing feelings about reading that from a environment variable since this is the way the ssh client handles it but all arguments used by mole is either passed as a CLI argument or read from the ssh config file.

So, to keep some consistency on how mole handles things, would you mind adding a new flag where the user can pass the location of the unix socket and add the signers if the flag is present and the socket exists?

Reply:
Flag added. Most often this will just be `--ssh-agent $SSH_AUTH_SOCK` but I can see the reasoning behind wanting a flag. Documentation updated as well.